test/Constraints/super_constructor.swift

```diff
@@ -55,12 +55,12 @@ class B {
 
 // SR-2484: Bad diagnostic for incorrectly calling private init
 class SR_2484 {
-  private init() {} // expected-note {{'init()' is inaccessible due to 'private' protection level}}
-  private init(a: Int) {} // expected-note {{'init(a:)' is inaccessible due to 'private' protection level}}
+  private init() {} // expected-note {{'init' declared here}}
+  private init(a: Int) {} // expected-note {{'init' declared here}}
```

Comment:
I thought that at first, but then I realized that the "new" output is just the general case diagnostics when any member lookup fails due to inaccessibility; and I didn't want to argue for changing the output of the general case. That wasn't the goal of this pull request.